scripts/insert_into_appointments.sql

```diff
@@ -1 +1,8 @@
 -- insert appointments creating connections between doctors and patients
+INSERT INTO appointments(appt_date, time, comment, patient_id, doctor_id)
+VALUES('2016-08-12', '10am',, 8, 4),
+      ('2016-08-17', '1pm',, 5, 6),
+      ('2016-08-12', '10am',, 8, 6),
+      ('2016-08-17', '1pm',, 7, 4)
```

Comment:
Hard-coding values like this will work, so I won't take points off for it, but it probably would have been a better idea to look up the specific patients and doctors from their respective tables.